### PR TITLE
working on conflicting file change with my team mate

### PR DESCRIPTION
**Pull request by MadinaDanaeva**
added changes on hello2.java

### CODE REVIEW

**Comment by YigitTolga** [comment]
whats the conflic?  :D

**Review by zizuasme33 — APPROVED** [review]
Hope this conflict will resolve..

File: src/test/java/Madina/hello2.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package Madina;
+
+public class hello2 {
+    //awesome cucumber stuff here
+}
```
Comment (YigitTolga) [inline src/test/java/Madina/hello2.java:6]: I didnt understand that conflic stuff :(

**Review by YigitTolga — CHANGES_REQUESTED** [review]
I did nt get conflic part.

**Comment by YigitTolga** [comment]
like what i get. If 2 teammate
 work on the same line and push. is it became a conflic?

**Comment by tumacho-hub** [comment]
I'm  closing i guess

**Pull request closed without merging**